pkg/api/dashboard_snapshot.go

```diff
@@ -357,8 +167,13 @@ func (hs *HTTPServer) DeleteDashboardSnapshot(c *contextmodel.ReqContext) respon
 		return response.Error(http.StatusNotFound, "Failed to get dashboard snapshot", nil)
 	}
 
+	// TODO: introduce orgID check
```

Comment:
@evictorero -- lets add this in a follow up PR, the tests all pass locally, but not in in CI so I think there must be a difference in the org setup.  The fix will be subtle enough that it should not be included here